### PR TITLE
Move part of code into sdc-client library

### PR DESCRIPTION
**Pull request by AntonZelenin**

### CODE REVIEW

File: agent/src/agent/pipeline/pipeline.py [hunk 1]
```diff
@@ -250,9 +252,23 @@ def delay(self) -> str:
     def batch_size(self) -> str:
         return self.config.get('batch_size', 1000)
 
-    def set_streamsets(self, streamsets):
-        self.streamsets_id = streamsets.id
-        self.streamsets = streamsets
+    def get_config(self) -> dict:
```
Comment (dariakharlan) [inline agent/src/agent/pipeline/pipeline.py:255]: This name `get_config` might be confusing because the pipeline entity has a property `config`

**Review by dariakharlan — CHANGES_REQUESTED** [review]

File: agent/src/agent/pipeline/pipeline.py (hunk 1 above)
Comment (AntonZelenin) [reply agent/src/agent/pipeline/pipeline.py:255]: ok, I'll rename it here as well as in the IPipeline in the lib, just didn't want to give a long name to the interface method

File: agent/src/agent/pipeline/manager.py [hunk 2]
```diff
@@ -327,13 +330,23 @@ def get_sample_records(pipeline_: Pipeline) -> (list, list):
...
-        streamsets.manager.delete(test_pipeline)
+        sdc_client.delete(test_pipeline)
     return preview_data, errors
+
+
+def create_streamsets_pipeline_config(pipeline_: Pipeline) -> dict:
+    return get_config_handler(pipeline_).override_base_config()
+
+
+def _get_config_loader(pipeline_: Pipeline):
```
Comment (dariakharlan) [inline agent/src/agent/pipeline/manager.py:349]: Don't you have this method in agent/src/agent/pipeline/config/handlers/factory.py?

**Review by dariakharlan — CHANGES_REQUESTED** [review]
agent/src/agent/pipeline/config/base_pipelines/Monitoring.json
agent/src/agent/pipeline/config/handlers/monitoring.py
these files should've been deleted, probably merge went wrong

Also you should remove all mentions of Monitoring pipeline from sdc-client

**Pull request merged**